### ISSUE TITLE
Cannot read property 'match' of undefined

### ISSUE THREAD

**Issue by ostraat** [hadynz/obsidian-kindle-plugin#113]
- Updated to version 1.4.1.
- Moved all previously synced files out of my vault.

5 books now won't sync (they worked a couple weeks ago). "5 books(s) could not be synced because of errors"

The Developer Tools Console shows 5 errors with the same `TypeError`:

```
Error syncing book
```
```json
{
    "id": "61164",
    "asin": "B01HAKH474",
    "title": "How to Get Unstuck: Breaking Free from Barriers to Your Productivity",
    "author": "Matt Perman and Matthew Aaron Perman",
    "url": "https://www.amazon.com/dp/B01HAKH474",
    "imageUrl": "https://m.media-amazon.com/images/I/81zRvThih2L._SY160.jpg",
    "lastAnnotatedDate": "2019-09-09T05:00:00.000Z"
}
```
```
TypeError: Cannot read property 'match' of undefined
    at eval (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:986558)
    at eval (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:986614)
    at Array.map (<anonymous>)
    at Vt (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:986434)
    at async Xt (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:986997)
    at async kn.syncBooks (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:998940)
    at async kn.startSync (eval at <anonymous> (app.js:formatted:62030), <anonymous>:2:998266)
```

**Comment by rphillips** [comment]
I got this same error as well.

```
asin: "B0010SKUYC"
author: "Brad Thor"
id: "27892"
imageUrl: "https://m.media-amazon.com/images/I/71e5oEnGI3L._SY160.jpg"
lastAnnotatedDate: Fri Dec 30 2011 00:00:00 GMT-0600 (Central Standard Time) {}
title: "The Last Patriot: A Thriller (The Scot Harvath Series Book 7)"
url: "https://www.amazon.com/dp/B0010SKUYC"
```

**Comment by hadynz** [comment]
This issue should be fixed in the latest plugin version 1.5.2. Can you try it out and confirm if it has resolved your issue please?

**Comment by ostraat** [comment]
Yes, 1.5.2 resolves my issue.  Thanks!
